CMakeLists.txt

```diff
@@ -46,6 +46,10 @@ if(WIN32)
 	)
 	target_link_libraries(${PROJECT_NAME} wsock32 ws2_32 -static-libgcc -static-libstdc++)
 	file(COPY assets DESTINATION ${CMAKE_BINARY_DIR}/bin)
+if(APPLE)
+  target_compile_options(${PROJECT_NAME} PRIVATE -w -O3 -s -Wno-narrowing)
```

Comment:
```suggestion
```

No need to duplicate already defined `-w -O3 -s` options here, although it's not critical. 
I suggest moving `-Wno-narrowing` to [common project compile options](https://github.com/regular-dev/biplanes-revival/blob/eb7e4323dd72d68ee1f0ef368b267ae90c70cd67/CMakeLists.txt#L35). Back then I preferred turning it on in IDE